versions/3.0.md

```diff
@@ -786,9 +734,89 @@ schema:
   type: file
 ```
 
+#### <a name="requestBodyObject"></a>Request Body Object
+
+Describes a single request body.
+
+If the `requestBody` attribute is not present, or `schema` is not defined, the request body is not required.
```

Comment:
Adding `required` back in. 

I'm specifying that it defaults to `true`, to avoid having to specify `required: true` for every request body (when that is probably the 90+% case).